internal/identity/okta.go

```diff
@@ -74,8 +74,6 @@ func NewOktaProvider(p *Provider) (*OktaProvider, error) {
 // https://developer.okta.com/docs/api/resources/oidc#revoke
 func (p *OktaProvider) Revoke(ctx context.Context, token *oauth2.Token) error {
 	params := url.Values{}
-	params.Add("client_id", p.ClientID)
```

Comment:
Yeah, I did a bit more digging, and the RFC for revocation (though not part of OIDC standard) should not contain CLIENTID /Secret

https://tools.ietf.org/html/rfc7009

Reply:
I read the resource, nothing concrete about the implementation but I hope most identity providers will handle it in the generic way we're implementing it.

Reply:
Actually, this is required. https://developer.okta.com/docs/reference/api/oidc/#client-secret